pkg/kubelet/kubelet.go

```diff
@@ -1805,6 +1805,21 @@ func (kl *Kubelet) syncLoop(updates <-chan kubetypes.PodUpdate, handler SyncHand
 		if !kl.syncLoopIteration(updates, handler, syncTicker.C, housekeepingTicker.C, plegCh) {
 			break
 		}
+
+		kl.syncLoopMonitorCheck()
```

Comment:
@yujuhong 

> How about moving the kl.syncLoopMonitor operations to this loop instead? This way, it gets updated even if runtime is not functioning.

There are two update to thee `syncLoopMonitor`. One is at he start of `syncLoopIteration`, the other is at the end of `syncLoopIteration`. I am not quite sure why we need the two updation. Maybe the end update of `syncLoopMonitor` is to indicate the long running of sync operation. 
If that is true, we may need to do this:
```golang
		kl.syncLoopMonitor.Store(kl.clock.Now())
		if !kl.syncLoopIteration(updates, handler, syncTicker.C, housekeepingTicker.C, plegCh) {
			break
		}
		kl.syncLoopMonitor.Store(kl.clock.Now())
```

> I am not sure why you moved the check here.

The intention is to remove sync loop healthz endpoint and instead just logging this situation. :)